etl/steps/data/garden/un/2023-08-02/comtrade_pandemics.py

```diff
@@ -124,6 +124,8 @@ def run(dest_dir: str) -> None:
     tb = add_regions(tb, ds_regions)
 
     # Correct former country names
+    tb.country = tb.country.astype(str)
```

Comment:
I think this plays nicer:

```suggestion
    tb.country = tb.country.astype("string")
```